pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/stats/MetadataStoreStats.java

```diff
@@ -24,41 +24,42 @@
 
 public final class MetadataStoreStats implements AutoCloseable {
     private static final double[] BUCKETS = new double[]{1, 3, 5, 10, 20, 50, 100, 200, 500, 1000, 2000, 5000};
-    private static final String LABEL_NAME = "metadata_store_name";
+    private static final String LABEL_NAME = "name";
+    protected static final String PREFIX = "pulsar_metadata_store_";
 
     private static final Histogram GET_OPS_SUCCEED = Histogram
-            .build("pulsar_metadata_store_get_ops_latency", "-")
+            .build(PREFIX + "get_latency", "-")
             .unit("ms")
             .buckets(BUCKETS)
             .labelNames(LABEL_NAME)
             .register();
     private static final Histogram DEL_OPS_SUCCEED = Histogram
-            .build("pulsar_metadata_store_del_ops_latency", "-")
+            .build(PREFIX + "del_latency", "-")
```

Comment:
`del_latency` --> `del_latency_ms` (according to [this](https://prometheus.io/docs/practices/naming/))

Reply:
I think unit "ms" will append _ms automatically?

Reply:
I'm thinking how about adding operation type as a label?

It looks like 

pulsar_metadata_store_op_latency_ms{type="put/get/del"}
pulsar_metadata_store_op_failed_total{type="put/get/del"}

Reply:
I'm not very sure about this part. Just the first impression, using labels here will reduce the metrics name.
Documentation will also be more concise.

Reply:
I don't think so, it just changes
```
pulsar_metadata_store_get_latency_ms{name = 'xx'}
pulsar_metadata_store_del_latency_ms{name = 'xx'}
pulsar_metadata_store_put_latency_ms{name = 'xx'}
```
to
```
pulsar_metadata_store_latency_ms{name = 'xx', type = 'get'}
pulsar_metadata_store_latency_ms{name = 'xx', type = 'del'}
pulsar_metadata_store_latency_ms{name = 'xx', type = 'put'}
```
It comes no benefits and makes metrics data longer.

Reply:
The advantage you get is mostly when you want to look at those metrics together - like give me an average of all operations. Is it something we'll need or always look at per action?

Reply:
I don't think that `an average of all operations` is meaningful, I'll ask prometheus community which is better.

Reply:
@codelipenghui   https://prometheus.io/docs/practices/naming/   you're right, Prometheus recommend using `label` to distinguish